### DISCUSSION TITLE
LineChart添加了2条Serie并且都有实时更新数据，但只绘出来1条曲线是怎么回事

### DISCUSSION THREAD

**Discussion by gyp199009** [XCharts-Team/XCharts#264] [General]
版本3.5.0 ，xaxis的type是value，两组数据同时更新，而且值完全不一样，一个正弦，一个余弦，但是最后只有一条线画出来，如果把xaxis的type改为category就可以画2条线，这是为什么呢

**Comment by monitor1394** [comment]
啥资料信息都没有，没法知道哪的问题。

**Comment by monitor1394** [comment]
方便的话，给一下测试用例，可以Q群里发给【XCharts技术支持】
